contracts/src/v0.7/ChainlinkClient.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.7.0;
 import "./Chainlink.sol";
```

Comment:
What's the plan for porting these changes backwards and forwards?

Reply:
Get the reference version in and then copy the changes over in a separate PR.